scriptmodules/emulators/ti99sim.sh

```diff
@@ -36,7 +36,7 @@ function install_ti99sim() {
 function configure_ti99sim() {
     mkRomDir "ti99"
     moveConfigDir "$home/.ti99sim" "$md_conf_root/ti99/"
+    ln -sf "$biosdir/TI-994A.ctg" "$md_inst/TI-994A.ctg"
```

Comment:
guess I'm still stuck on legacy functions (eg. gba script as well has it etc.) will fix